fpa/grpc/service.go

```diff
@@ -138,12 +161,28 @@ func (s *Service) AddFile(srv pb.API_AddFileServer) error {
 
 	go receiveFile(srv, writer)
 
-	cid, err := i.AddFile(srv.Context(), reader)
+	c, err := s.hot.Add(srv.Context(), reader)
+	if err != nil {
+		return fmt.Errorf("adding data to hot layer: %s", err)
+	}
+
+	jid, err := i.AddCid(c)
 	if err != nil {
 		return err
 	}
+	ch, err := i.Watch(jid)
```

Comment:
So here, we're blocking and watching the job internally, but in the future the caller would receive these job events?

Reply:
Yep, now does this internally just to not change the gRPC APIs and CLI... but after having a more stable internal API, sound better to let the client watch to whatever might be interested.